go/vt/proto/automationservice/automationservice.pb.go

```diff
@@ -12,7 +12,6 @@ import (
 	grpc "google.golang.org/grpc"
 	codes "google.golang.org/grpc/codes"
 	status "google.golang.org/grpc/status"
-
```

Comment:
it looks like the new `goimports` style conflicts with how `protoc-gen-go` generates files

Reply:
Fixes for this in 602dd69 (git hook) and 67ae893 (linter config)